.github/workflows/no-response.yml

```
@@ -10,8 +10,8 @@ on:
   schedule:
     # These runs in our repos are spread evenly throughout the day to avoid hitting rate limits.
     # If you change this schedule, consider changing the remaining repositories as well.
-    # Runs at 12 am, 12 pm
-    - cron: '0 0,12 * * *'
+    # Runs at 1 am, 1 pm
+    - cron: '0 0, 1 * * *'
```

That's the correct expression for 1am, 1pm (see https://crontab.guru/#0_1,13_*_*_*):

```suggestion
    - cron: '0 1,13 * * *'
```